scripts/install

```diff
@@ -157,10 +157,9 @@ ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$ap
 ynh_script_progression --message="Configuring SSOwat..." --weight=1
 
 # Make app public if necessary
-if [ $is_public -eq 1 ]
+if [ $is_public -eq 0 ]
 then
-	# unprotected_uris allows SSO credentials to be passed anyway.
-	ynh_app_setting_set --app=$app --key=unprotected_uris --value="/"
+	ynh_permission_update --permission "main" --remove "visitors"
 fi
```

Comment:
I'm not sure if we need this permission here ... maybe redundant

Reply:
It seems fine to me, what would it be redundant with?

Reply:
As `$is_public` cannot be set to `1`, `0` is the default 

The CI warnings seem to suggest this ...
```
Warning: yunohost.user user_permission_update - [4357.1] Group 'visitors' already has permission 'nodered.main' disabled
Warning: yunohost.user user_permission_update - [4357.1] The permission was not updated because the addition/removal requests already match the current state.
Warning: yunohost.hook <lambda> - [1038.1] Group 'visitors' already has permission 'nodered.main' disabled
Warning: yunohost.hook <lambda> - [1038.1] The permission was not updated because the addition/removal requests already match the current state.
```

Reply:
Ah, I did not see those warnings. Though you followed `example_ynh`.... did something change about permissions handling in v4.0.x?

Reply:
Ah no, I'm tired. I forgot I completely removed `is_public` from the manifest. We can remove it then, I guess.
It makes me think we could add a `ui.nodered` permission, to handle the `/ui` path when user add the dashboard plugin.